cli/lib/kontena/plugin_manager/loader.rb

```diff
@@ -41,12 +41,13 @@ def load_plugin(path, spec)
         Kontena.logger.debug { "Loaded plugin #{spec.name}" } if plugin_debug?
 
         report_tracking
-        loaded_plugins << spec
         true
-      rescue LoadError, ScriptError => ex
+      rescue => ex
```

Comment:
Need to keep the `LoadError, ScriptError`, because they are not of `StandardError`

```
$ bin/kontena plugin list
/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require': cannot load such file -- droplet_kit (LoadError)
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require'
	from /home/kontena/.kontena/gems/2.3.1/gems/kontena-plugin-digitalocean-0.3.2/lib/kontena_cli_plugin.rb:4:in `<top (required)>'
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require'
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require'
	from /home/kontena/kontena/kontena/cli/lib/kontena/plugin_manager/loader.rb:40:in `load_plugin'
	from /home/kontena/kontena/kontena/cli/lib/kontena/plugin_manager/loader.rb:21:in `block in load_plugins'
	from /home/kontena/kontena/kontena/cli/lib/kontena/plugin_manager/loader.rb:17:in `each'
	from /home/kontena/kontena/kontena/cli/lib/kontena/plugin_manager/loader.rb:17:in `load_plugins'
	from /home/kontena/kontena/kontena/cli/lib/kontena/plugin_manager.rb:22:in `plugins'
	from /home/kontena/kontena/kontena/cli/lib/kontena/plugin_manager.rb:15:in `init'
```